### PR TITLE
Docs:Move searchbar to the top of the page and Widen the Search Preview Window(fix #4095)

### PR DESCRIPTION
**Pull request by kyle-cx**
fix issue https://github.com/apache/iceberg/issues/4095

it's hard to widen the result view below the search input, so I put it on the right side
@RussellSpitzer

### CODE REVIEW

**Comment by RussellSpitzer** [comment]
Could you please attach a image showing the change?

**Comment by RussellSpitzer** [comment]
Oh I see them on the Issue, Looks great to me. @samredai wdyt?

**Review by RussellSpitzer — APPROVED** [review]
I like how it looks, but I don't know enough about HTML to know if this is the right way to do anything :)

**Comment by samredai** [comment]
Thanks @kyle-cx ! I think this looks great but since the search results have been moved to the base page, the results are relegated to the background on mobile when the left nav-bar collapses. We could have two result boxes and show one based on media queries, but that would require some changes to the javascript search code which might be better to leave alone.

Let me play around with this a bit and see if I can come up with a solution 🔧

**Comment by kyle-cx** [comment]
@samredai good idea. I didn't consider the problem of the mobile side. I planned to widen it on the left side, but the z-index problem between <aside> and base page bothered me for a long time. I'm not very good at js, so I'm sorry to say The rest is up to you

**Comment by kyle-cx** [comment]
@samredai cool! Never mind, glad to be forked

File: docs/themes/hugo-book/layouts/_default/baseof.html [hunk 1]
```diff
@@ -15,6 +15,13 @@
     </aside>
 
     <div class="book-page">
+      <div >
```
[the remark this answers is not in the document]
Comment (kyle-cx) [reply docs/themes/hugo-book/layouts/_default/baseof.html:18]: thanks for the advice

**Comment by kyle-cx** [comment]
hi @samredai, anything more I need to do for this ?

**Comment by samredai** [comment]
@kyle-cx LGTM!

cc: @RussellSpitzer @rdblue

**Pull request merged**